core-services/workflow/src/main/java/org/silverpeas/core/workflow/engine/AbstractRequest.java

```diff
@@ -0,0 +1,47 @@
+package org.silverpeas.core.workflow.engine;
+
+import org.silverpeas.core.thread.task.AbstractRequestTask;
+import org.silverpeas.core.workflow.api.WorkflowException;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'org.silverpeas.core.workflow.api.WorkflowException'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AUselessImportCheck)